test/e2e/index.test.ts

```diff
@@ -276,6 +276,38 @@ describe('e2e', () => {
 
       expect(pageErrors.length).to.equal(0);
     });
+
+    it('should not glitch when resizing', async () => {
+      await renderFixture('TextareaAutosize/BasicTextareaAutosize');
+
+      const textarea = await screen.getByTestId('textarea')!;
+
+      // Get the element's dimensions
+      const { x, y, width, height } = (await textarea.boundingBox())!;
+
+      // Calculate coordinates of bottom-right corner
+      const bottomRightX = x + width;
+      const bottomRightY = y + height;
+
+      // Get the initial height of textarea as a number
+      const initialHeight = await textarea.evaluate((event) => parseFloat(event.style.height));
+
+      // Move the mouse to the bottom-right corner, adjusting slightly to grab the resize handle
+      await page.mouse.move(bottomRightX - 5, bottomRightY - 5);
+
+      // Simulate a double click without releasing the mouse button (mouseup) to grab the resize handle
+      await page.mouse.down();
+      await page.mouse.up();
+      await page.mouse.down();
+
+      // Move the mouse to resize the textarea
+      await page.mouse.move(bottomRightX + 50, bottomRightY + 50);
+
+      // Assert that the textarea height has increased after resizing
+      expect(await textarea.evaluate((event) => parseFloat(event.style.height))).to.be.greaterThan(
```

Comment:
Why call the handle `event`? Something like `textareaHandle` would be easier to understand IMO.

Reply:
I'm not sure what you mean by `textareaHandle`. But you're right, it shouldn't be `event`. I've updated it to `textareaElement`. According to the documentation here: https://playwright.dev/docs/next/api/class-locator#locator-evaluate, `pageFunction` takes the element as an argument.